scylla.go

```diff
@@ -353,7 +353,7 @@ func newScyllaConnPicker(conn *Conn) *scyllaConnPicker {
 	}
 }
 
-func (p *scyllaConnPicker) Pick(t Token, keyspace string, table string, isLWT bool) *Conn {
+func (p *scyllaConnPicker) Pick(t Token, qry ExecutableQuery) *Conn {
 	if len(p.conns) == 0 {
 		return nil
 	}
```

Comment:
passing `nil` in `scylla_test.go` only works because token is `nil` in these tests and `Pick` execution finished here: https://github.com/scylladb/gocql/blob/master/scylla.go#L361. If token would be something different it would result in a `nil` pointer dereference

Reply:
Nulls are truly the largest footgun in IT.

Reply:
That's why it's very surprising that a new language like Go decided to keep them...

Reply:
I added additional validation

Reply:
But shouldn't we explicitly check against `nil` in `qry`, just like we check in `t`?
```go
if t == nil || qry == nil {
    return p.leastBusyConn()
}
```

Reply:
I am not sure, are you okay with my change or should I check for `nil` in different place?

Reply:
It seems OK now.